lib/elixir/lib/port.ex

```diff
@@ -3,56 +3,67 @@ defmodule Port do
   Functions related to Erlang ports.
   """
 
+  @type name :: {:spawn, string | binary} |
+                {:spawn_driver, string | binary} |
+                {:spawn_executable, string | atom} |
+                {:fd, non_neg_integer, non_neg_integer}
+
   @doc """
-  See [`:erlang.open_port/2`](http://www.erlang.org/doc/man/erlang.html#open_port-2)
+  Opens an Erlang port given a tuple `name` and a list of settings.
 
-  Inlined by the compiler.
```

Comment:
@lexmag I didn't know it was important to keep this sentence. I removed every occurence of them. Should I put them back?